blog.qmd

```diff
@@ -0,0 +1,177 @@
+---
+title: "Unified Branding Across Posit Tools with brand.yml"
+format: html
+brand: false
+navbar: false
+---
+
+## Why **brand.yml**?
+
+### Consistent branding is difficult 
+
+Consistent branding ensures your work looks professional, cohesive, and polished, but applying brand guidelines can be challenging. Your brand guidelines might include detailed requirements for logos, colors, and fonts. Handling branding separately for each product or output takes time and effort, and small inconsistencies can add up.
+
+Other common challenges with traditional approaches to branding include:
+
+* **Excessive manual effort**: Applying colors, fonts, and logos across reports, apps, and dashboards takes time and is prone to errors.
+
+* **Difficult to update**: When brand guidelines change, it’s difficult to update all products consistently.
+
+* **Team consistency**: Ensuring all contributors follow branding guidelines across reports, dashboards, and apps can be challenging to manage. 
+
+### The solution: **brand.yml**
+
+Enter [**brand.yml**](https://posit-dev.github.io/brand-yml/). We created **brand.yml** to simplify the process of applying branding guidelines to your work. With **brand.yml**, you can define your organization's branding in a single YAML file and apply that branding across reports, dashboards, and apps created with Posit’s open-source tools.
+
+### Advantages of **brand.yml**
+
+**brand.yml** allows you to collect a brand's logos, colors, fonts, and typographic choices into a single `_brand.yml` file, which you can then apply instantly across Quarto reports and Shiny apps. 
+
+Here are some advantages of this strategy:
+
+* **Consistency without effort** Ensure all your work consistently follows brand guidelines and avoid duplication by defining branding in one file that can be applied across projects.
+
+* **Easy collaboration** Share a single `_brand.yml` file across your organization to ensure everyone can easily apply the same branding, eliminating the need for contributors to recreate brand-specific designs for every project.
+
+* **Not just for corporate branding** You can also use **brand.yml** to create custom themes for your personal or team-specific projects. Create a `_brand.yml` file for a specific journal or conference to ensure your work matches their style. 
+
+## See **brand.yml** in action
+
+Let's see how **brand.yml** works. The first step is to define your brand elements, including logos, colors, fonts, and typography, in a single `_brand.yml` file. 
+
+Below is an example `_brand.yml` file that defines a logo, color palette, and font and typographic choices for an example brand. Learn more about the structure of a `_brand.yml` file [here](https://posit-dev.github.io/brand-yml/brand/).
+
+
+```yaml
+logo: 
```

Comment:
I’m partial to including `meta` here, even if it’s just `meta.name`.

Reply:
done! Thanks @gadenbuie